Makefile

```diff
@@ -235,6 +235,10 @@ golangci-lint-fix:
         $(MAKE) --no-print-directory -C $$mod golangci-lint-fix; \
     done
 
+.PHONY: test-gherkin-lint
+test-gherkin-lint:
+	gherlint tests/acceptance/features -c tests/acceptance/config/.gherlintrc.json
+
```

Comment:
we could also add `test-gherkin-lint-fix` command

Reply:
Sure.I'm on it.